CMakeLists.txt

```diff
@@ -49,6 +49,15 @@ endif ()
 
 set (CMAKE_CXX_STANDARD 11)
 
+# Use C++11
+if (CMAKE_VERSION VERSION_LESS "3.1")
+    if (CMAKE_CXX_COMPILER_ID STREQUAL "GNU")
+        set (CMAKE_CXX_FLAGS "--std=gnu++11 ${CMAKE_CXX_FLAGS}")
```

Comment:
Oh, I see.  No big deal I guess, as far as I can tell we're not using any new features.  But it's good to preemptively use newer standards if possible.